client/src/app/hooks/table-controls/sorting/getLocalSortDerivedState.ts

```diff
@@ -51,22 +52,10 @@ export const getLocalSortDerivedState = <
 
   let sortedItems = items;
   sortedItems = [...items].sort((a: TItem, b: TItem) => {
-    let aValue = getSortValues(a)[activeSort.columnKey];
-    let bValue = getSortValues(b)[activeSort.columnKey];
-    if (typeof aValue === "string" && typeof bValue === "string") {
-      aValue = aValue.replace(/ +/g, "");
-      bValue = bValue.replace(/ +/g, "");
-      const aSortResult = aValue.localeCompare(bValue, i18n.language);
-      const bSortResult = bValue.localeCompare(aValue, i18n.language);
-      return activeSort.direction === "asc" ? aSortResult : bSortResult;
-    } else if (typeof aValue === "number" && typeof bValue === "number") {
-      return activeSort.direction === "asc" ? aValue - bValue : bValue - aValue;
-    } else {
-      if (aValue > bValue) return activeSort.direction === "asc" ? -1 : 1;
-      if (aValue < bValue) return activeSort.direction === "asc" ? -1 : 1;
-    }
-
-    return 0;
+    const aValue = getSortValues(a)[activeSort.columnKey];
+    const bValue = getSortValues(b)[activeSort.columnKey];
+    const compareValue = universalComparator(aValue, bValue, i18n.language);
```

Comment:
Looks like this is missing the space collapsing that was there before. Does that change any thing?